client/src/main/java/org/apache/celeborn/client/compress/Lz4Decompressor.java

```diff
@@ -17,8 +17,13 @@
 
 package org.apache.celeborn.client.compress;
 
+import java.util.Map;
+import java.util.function.Supplier;
 import java.util.zip.Checksum;
 
+import scala.Option;
+
+import com.google.common.collect.ImmutableMap;
```

Comment:
This requires adding a line break to separate it from the following "net" statement.

Reply:
@kerwin-zk, I used `mvn spotless:apply -Pgoogle-mirror` to reformat and there is no line break.